src/mono/mono/profiler/mono-profiler-coverage.suppression

```diff
@@ -86,7 +86,6 @@ System.Interactive.Async
 System.Interactive.Providers
 System.IO.Compression
 System.IO.Compression.FileSystem
-System.Json
 System.Json.Microsoft
```

Comment:
What is System.Json.Microsoft?

Reply:
It's from a time when System.Json was still implemented using Mono code and System.Json.Microsoft contained some early open-sourced version that was included in the aspnet stack: https://github.com/mono/mono/tree/master/mcs/class/System.Json.Microsoft.

This file probably shouldn't have been touched since it contains only assemblies from mono/mono, I'll revert that change.

Reply:
> This file probably shouldn't have been touched since it contains only assemblies from mono/mono, I'll revert that change.

Thanks. Can we remove this and anything else mono/mono-specific from this repo?

Reply:
Not easily at the moment. I'll keep it in mind for future cleanups.